test/protobuf/from_multiple_files_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Protobuf.FromMultipleFiles.Test do
+  use Protobuf.Case
+
+  test "generates all messages nested under TopLevel" do
+    defmodule TopLevel do
+      use Protobuf, from: [Path.expand("../proto/basic.proto", __DIR__),
+                           Path.expand("../proto/import.proto", __DIR__),
+                           Path.expand("../proto/imported.proto", __DIR__)]
+    end
+
+    assert %{reason: "hi"} = TopLevel.Authorization.WrongAuthorizationHttpMessage.new(reason: "hi")
```

Comment:
Using `assert` with a single `=` will attempt a pattern match and give a nice error message on failure. For instance if I change that test to `assert %{reason: "hi", failure: "none"} = ...` then I get the following output on a test run.

```
  1) test generates all messages nested under TopLevel (Protobuf.FromMultipleFiles.Test)
     test/protobuf/from_multiple_files_test.exs:4
     match (=) failed
     code: %{reason: "hi", failure: "none"} = TopLevel.Authorization.WrongAuthorizationHttpMessage.new(reason: "hi")
     rhs:  %Protobuf.FromMultipleFiles.Test.TopLevel.Authorization.WrongAuthorizationHttpMessage{reason: "hi"}
     stacktrace:
       test/protobuf/from_multiple_files_test.exs:11
```

In this case I didn't specifically care about testing that it returned the right struct, I'm mostly just testing the fact that the module was generated by using its `new/1` function. That's why I went with a simplistic pattern match rather than a `==` comparison.